ui/app/controllers/jobs/run/templates/manage.js

```diff
@@ -23,7 +24,7 @@ export default class JobsRunTemplatesManageController extends Controller {
   @task(function* (model) {
     try {
       yield model.destroyRecord();
-
+      getOwner(this).lookup('route:jobs.run.templates.manage').refresh();
```

Comment:
Funky! Guessing this is our reactivity workaround. Thankfully we have solid testing around this so we're covered on maintainability.

Reply:
Ended up nixing this, but FWIW this is an established pattern from the Evaluations controller's refresh() method. Screams "global state editing" to me but ¯\_(ツ)_/¯

Reply:
Evaluations has a Refresh button in the UI because that view isn't real-time. We wanted to give users a fast way to clear all filters and refresh the page. We added the logic in when a user wanted to click a button to `refresh` the page per the designs. Evaluations was designed with reactivity in mind, however! That's why you see the statecharts! 😄 

```
  @action
  refresh() {
    const isDefaultParams = this.nextToken === null && this.status === null;
    if (isDefaultParams) {
      getOwner(this).lookup('route:evaluations.index').refresh();
      return;
    }

    this._resetTokens();
    this.status = null;
    this.pageSize = this.userSettings.pageSize;
  }
```